### ISSUE TITLE
Multi value custom field cannot be parsed

### ISSUE THREAD

**Issue by epic-alex-rodionov** [Crim/pardot-java-client#50]
Custom fields can have Multi-Select type and contain > 1 value.
In this case we see:
```
"error": "Cannot deserialize instance of java.lang.String out of START_OBJECT token\n at [Source: (StringReader); line: 54, column: 10] (through reference chain: com.darksci.pardot.api.response.prospect.ProspectReadResponse["prospect"]->com.darksci.pardot.api.response.prospect.Prospect["Add_Prospect01"])"
```
Root cause:
https://github.com/Crim/pardot-java-client/blob/master/src/main/java/com/darksci/pardot/api/response/prospect/Prospect.java#L125
src/main/java/com/darksci/pardot/api/response/prospect/Prospect.java:125
    `private Map<String, String> customFields = new HashMap<>();`

**Comment by epic-alex-rodionov** [comment]
I have fixed it locally. But I do not have permission to create pull request

**Comment by Crim** [comment]
If you fork the repository and commit the changes to your fork, you should be able to submit a pull request.

[this](https://jarv.is/notes/how-to-pull-request-fork-github/) should outline the process.  

Let me know if you have any questions or run into trouble, I look forward to your contribution! :)

**Comment by epic-alex-rodionov** [comment]
PR: https://github.com/Crim/pardot-java-client/pull/51

**Comment by Crim** [comment]
I released 2.0.0 which should include a fix for this (see #52) Let me know if that works for you.

Thanks!

**Comment by epic-alex-rodionov** [comment]
Thank you!
